saleor/graphql/account/mutations.py

```diff
@@ -617,7 +616,7 @@ class Arguments:
         )
 
     class Meta:
-        description = dedent(
+        description = (
```

Comment:
Brackets were left here on purpose? With multiline strings, we don't actually need them.

Reply:
Good point, I was trying to check _fast_ if this fixes the problem.